packages/typescript/test/test.js

```diff
@@ -1085,50 +1085,48 @@ test('supports custom transformers', async (t) => {
 });
 
 function fakeTypescript(custom) {
-  return Object.assign(
-    {
-      sys: ts.sys,
-      createModuleResolutionCache: ts.createModuleResolutionCache,
-      ModuleKind: ts.ModuleKind,
-
-      transpileModule() {
-        return {
-          outputText: '',
-          diagnostics: [],
-          sourceMapText: JSON.stringify({ mappings: '' })
-        };
-      },
-
-      createWatchCompilerHost() {
-        return {
-          afterProgramCreate() {}
-        };
-      },
-
-      createWatchProgram() {
-        return {};
-      },
-
-      parseJsonConfigFileContent(json, host, basePath, existingOptions) {
-        return {
-          options: {
-            ...json.compilerOptions,
-            ...existingOptions
-          },
-          fileNames: [],
-          errors: []
-        };
-      },
-
-      getOutputFileNames(_, id) {
-        return [id.replace(/\.tsx?/, '.js')];
-      },
-
-      // eslint-disable-next-line no-undefined
-      getTsBuildInfoEmitOutputFilePath: () => undefined
+  return {
+    sys: ts.sys,
+    createModuleResolutionCache: ts.createModuleResolutionCache,
+    ModuleKind: ts.ModuleKind,
+
+    transpileModule() {
+      return {
+        outputText: '',
+        diagnostics: [],
+        sourceMapText: JSON.stringify({ mappings: '' })
+      };
     },
-    custom
-  );
+
+    createWatchCompilerHost() {
+      return {
+        afterProgramCreate() {}
+      };
+    },
+
+    createWatchProgram() {
+      return {};
+    },
+
+    parseJsonConfigFileContent(json, host, basePath, existingOptions) {
+      return {
+        options: {
+          ...json.compilerOptions,
+          ...existingOptions
+        },
+        fileNames: [],
+        errors: []
+      };
+    },
+
+    getOutputFileNames(_, id) {
+      return [id.replace(/\.tsx?/, '.js')];
+    },
+
+    // eslint-disable-next-line no-undefined
+    getTsBuildInfoEmitOutputFilePath: () => undefined,
+    ...custom
```

Comment:
Do we have tests that cover `custom`? There are quite a few differences with `Object.assign` and `custom` so I want to make sure we're covered.

Reply:
I've reverted the changes to this file. I think the linting changes would better be handled as part of the discussion in https://github.com/rollup/plugins/issues/678